en/docs/try-out/custom-throttling-policies.md

```diff
@@ -0,0 +1,68 @@
+Custom throttling allows system administrators to define dynamic rules for specific use cases. When a custom throttling
+policy is created, it is possible to define any policy you like. This feature is required when certain traffic thresholds
+need to be freely throttled or rejected by Data Holders without impacting their performance or availability requirements.
```

Comment:
Let's remove all CDS related terms and sentences.

Reply:
Fixed in 6db3e9a73e9f3e9253d7588f25fdeb7c3d1a1e05.